src/styles/blocks/stars.scss

```
@@ -0,0 +1,43 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: url("../images/star.svg") center no-repeat;
+
+    &:not(:first-child) {
+      margin-left: 4px;
+    }
+  }
+}
+
+.stars--1 {
+  .stars__star:first-child {
+    background: url("../images/star-active.svg") center no-repeat;
```

```suggestion
    background-image: url("../images/star-active.svg");
```
don't duplicate other background properties